client/jetpack-cloud/sections/onboarding-tours/dashboard-walkthrough-tour/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { useTranslate } from 'i18n-calypso';
+import GuidedTour from 'calypso/jetpack-cloud/components/guided-tour';
+import { JETPACK_MANAGE_ONBOARDING_TOURS_PREFERENCE_NAME } from '../constants';
+
+import '../style.scss';
+
+export default function DashboardWalkthroughTour() {
+	const translate = useTranslate();
+
+	const urlParams = new URLSearchParams( window.location.search );
+	const shouldRenderDashboardTour = urlParams.get( 'tour' ) === 'dashboard-walkthrough';
+
+	return (
+		shouldRenderDashboardTour && (
+			<GuidedTour
+				className="onboardig-tours__guided-tour"
+				preferenceName={ JETPACK_MANAGE_ONBOARDING_TOURS_PREFERENCE_NAME[ 'dashboardWalkthrough' ] }
+				tours={ [
+					{
+						target: "a.section-nav-tab__link[tabindex='0']",
+						popoverPosition: 'bottom right',
+						title: translate( 'Manage all your sites' ),
+						description: translate(
+							'Here you can find your sites and detailed overview about each.'
+						),
+					},
+					{
+						target: '.site-table__table tr:first-child td.jetpack-cloud-site-column__stats',
+						popoverPosition: 'bottom right',
+						title: translate( '📊 7 days insights stats' ),
+						description: translate( 'Here you can see key metrics (visitors and views).' ),
+					},
+					{
+						target: '.site-table__table tr:first-child td.jetpack-cloud-site-column__boost',
+						popoverPosition: 'bottom right',
+						title: translate( '🚀 Boost score rating' ),
+						description: translate(
+							"Here's a score reflecting your website's load speed. Click 'Get Score' to know your site's speed rating – it's free!"
+						),
+					},
+					{
+						target: '.site-table__table tr:first-child td.jetpack-cloud-site-column__backup',
+						popoverPosition: 'bottom right',
+						title: translate( '🛡️ Backups' ),
+						description: translate(
+							'We automatically back up your site and safeguard your data. Restoring is as simple as a single click.'
+						),
+					},
+					{
+						target: '.site-table__table tr:first-child td.jetpack-cloud-site-column__scan',
+						popoverPosition: 'bottom right',
+						title: translate( '🔍 Scan' ),
+						description: translate(
+							'We scan your site and flag any detected issues using a traffic light warning system – 🔴 for severe or 🟡 for a warning.'
+						),
+					},
+					{
+						target: '.site-table__table tr:first-child td.jetpack-cloud-site-column__monitor',
+						popoverPosition: 'bottom right',
```

Comment:
It looks like this should be bottom left based on the Figma designs:
```suggestion
						popoverPosition: 'bottom left',
```

Reply:
Added: 591da1b9b9797bf9d45fc029cf5c887441a486c9